### PR TITLE
created march update post

### PR DESCRIPTION
**Pull request by rubymorillo**
Each month, we're cross-posting monthly updates from Ruby Together on the Bundler blog. I created the new post, based on the updates already published in Ruby Together.

### CODE REVIEW

File: source/conduct.html.md [hunk 1]
```diff
@@ -1,47 +1,136 @@
----
-title: Code of conduct
```
Comment (segiddins) [inline source/conduct.html.md]: do we no longer need the YAML front matter?

Comment (indirect) [reply source/conduct.html.md]: maybe the automatic generator didn’t catch the file for some reason? I’m not sure.

**Comment by indirect** [comment]
...and [this](https://travis-ci.org/bundler/bundler-site/builds/363672996?utm_source=github_status&utm_medium=notification) is why I should never use the merge button. blahhhhhh

**Comment by rubymorillo** [comment]
@indirect let me know if I need to change anything else!

**Pull request merged**